janus_server/src/aggregator/aggregate_share.rs

```diff
@@ -568,23 +569,26 @@ where
     for<'a> &'a A::AggregateShare: Into<Vec<u8>>,
 {
     // Check how many rows in the relevant table have an intersecting batch interval.
-    // Each such row consumes one unit of batch lifetime (§4.6).
-    let intersecting_intervals: Vec<_> = match task.role {
+    // Each such row consumes one unit of query count (§4.6).
```

Comment:
nit: I think this should refer to section [4.5.6](https://www.ietf.org/archive/id/draft-ietf-ppm-dap-02.html#name-batch-validation) or [4.5.7](https://www.ietf.org/archive/id/draft-ietf-ppm-dap-02.html#section-4.5.7) now. Though there's lots of incorrect section references in Janus.

Reply:
I removed the section reference entirely. I think it's only of marginal value given how fungible the DAP specification still is.

Maybe we can do a pass or something once per major version to update all of the section references appropriately? I don't think a process based on one-off updating section references as we touch the relevant code lines will work well -- even code that doesn't need to change to update the DAP version might well have its underlying spec sections moved around under it.

Reply:
What I've done most recently is include a link to the relevant spec section instead of the section number. Then at least it's a little more obvious that a reference is out of date, because you have the draft revision in the URL. However I'm also OK with not putting in section references until DAP stabilizes.